microraiden/microraiden/webui/microraiden/.gitignore

```diff
@@ -1,6 +0,0 @@
-node_modules/
```

Comment:
You removed the whole docs and a lot of other things from microraiden repo. Please, clean your changes, and from current master, be sure to only include your actual changes.